packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -227,9 +226,6 @@ const App = (props) => {
         })
     }, [])
 
-    // Handle updating the shopper context
-    useUpdateShopperContext()
```

Comment:
since the return value of the `useUpdateShopperContext` hook needs to be accessed in components we need to call this hook directly in the component it will be used